plugin/authorizer_client_gcp.go

```diff
@@ -30,7 +30,9 @@ func (c *gcpClient) InstanceGroups(ctx context.Context, project string, boundIns
 			for k, v := range l.Items {
 				zone, err := zoneFromSelfLink(k)
 				if err != nil {
-					return err
```

Comment:
I don't like swallowing this error. Can we check for a substring of `/regions` in `k` before calling `zoneFromSelfLink` instead?

```go
if strings.Contains(k, "/regions") {
  log.Printf("got back a region, dunno why")
  continue
}

zone, err := zoneFromSelfLink(k)
if err != nil {
  return err
}
```